data/json/mapgen/fema/FEMA_le_03.json

```diff
@@ -42,7 +42,7 @@
         "L": { "item": "surgery", "chance": 30 },
         "X": { "item": "hospital_lab", "chance": 80 }
       },
-      "item": { "B": { "item": "rag_bloody", "amount": [ 0, 5 ] } },
+      "item": { "B": { "item": "rag", "amount": [ 0, 5 ], "custom-flags": [ "FILTHY" ] } },
```

Comment:
"custom-flags" is not working here.

Reply:
what is the proper way to do it then?

Reply:
Add code that it would be supported in mapgen objects.

Reply:
This whole PR is completely JSON, seems to me that actual cpp edits should be handled outside of this

Reply:
I've added code to support this in #44362. After it is merged, you can rebase your PR on top of latest master and CI error should go away.